src/Google.Api.CommonProtos/TypeExtensions/Date.cs

```diff
@@ -0,0 +1,99 @@
+﻿/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using System;
+
+namespace Google.Type
+{
+    /// <summary>
+    /// Extensions to <see cref="Date"/>.
```

Comment:
Please remove the XML tag - these aren't extensions, as we're using a partial class, and we'd want the summary tag from the generated one to be used.